tools/codegen/corteza-api-client.js

```diff
@@ -194,6 +60,11 @@ namespaces.forEach(({ path, namespace, className }) => {
     throw err
   }
 
+  if (!spec) {
+    console.error('Endpoints are undefined')
+    return
```

Comment:
may I suggest process.exit(1)? If this was scripted and included in a bash script, the error code could be checked via $? . @darh

Reply:
sure.